DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -3205,9 +3205,12 @@ private static void UpdateFoldersForParentId()
         }
 
 
-        private static void UninstallPackage(string packageName, string packageType, bool deleteFiles = true)
+        private static void UninstallPackage(string packageName, string packageType, string version = "", bool deleteFiles = true)

```

Comment:
Ideally, adding the new parameter to the end of the method signature would be helpful to avoid the more cluttered code later in the process, as it would no longer require the use of a "string.empty" call in the version for the one package that was deleted